megalinter/descriptors/git.megalinter-descriptor.yml

```diff
@@ -24,4 +24,20 @@ linters:
     cli_help_arg_name: "--help"
     examples:
       - "git diff --check"
-
+  # GIT_PATHCHK
+  - linter_name: git_pathchk
+    lint_all_files: true
+    test_folder: git_pathchk
+    linter_text: Git files non-portable pathname check
+    linter_url: https://git-scm.com
+    linter_repo: https://github.com/git/git
+    linter_megalinter_ref_url: never
+    cli_config_arg_name: ""
+    cli_executable: /bin/sh
```

Comment:
I'd prefer executable to be git , and other args managed in cli_lint_extra_args, so that debugging in windows would work too, please could you manage that ? :)

Reply:
Yes, also my preference, but in this case not possible. a) It is  complex shell command, hence the shell to execute it. b) I have no Windows test-system and I assume despite Windows has some kind of Posix certification, the Posix Pathchk utility is not available on that platform. c) The Mega-Linter container is Linux.

Reply:
@nvuillam Btw, how can I execute the configured linker on bare-metal? Or do I need to build the 6-7 GB container first? This linter does not need any new binary, I can mount the repo into the container if it helps.

Reply:
> Plz also update CHANGELOG :)

I have no clue how that works. Do you have any documentation at hand?

Also do you have any developer / contributing documentation? Install from source? Develop the source version? Anything?

Reply:
Contributing guide is here :)
https://nvuillam.github.io/mega-linter/contributing/

If infos are missing, just ask ^^

Reply:
Thanks, I was missing this doc, that helps a lot.